docs/source/release/v3.8.0/framework.rst

```diff
@@ -51,6 +51,12 @@ Improved
 - :ref:`SavePlot1D <algm-SavePlot1D>` has options for writing out
   plotly html files.
 
+- :ref:`SofQW <algm-SofQW>` has option to replace any NaNs in output workspace
+  with zeroes.
+
+- :ref:`ConvertTableToMatrixWorkspace <algm-ConvertTableToMatrixWorkspace>` The input dialog
+  had a bug where the table columns were in a reversed order in the dialogue's combo boxes.
+
```

Comment:
Oops, is this paragraph repeated (see below)? That may have happened if there was a merge conflict in this release notes file.